source/code/plugin/health/monitor_factory.rb

```diff
@@ -0,0 +1,28 @@
+module HealthModel
+    class MonitorFactory
```

Comment:
what factory pattern is used here ?

Reply:
this is just a factory that returns a unit monitor or agg monitor.